telco-hub/configuration/reference-crs/required/gitops/addPluginsPolicy.yaml

```diff
@@ -0,0 +1,133 @@
+kind: ManagedClusterSetBinding
+metadata:
+  name: default
+  namespace: local-cluster
+  annotations:
+    argocd.argoproj.io/sync-wave: "5"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  clusterSet: default
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  annotations:
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+    policy.open-cluster-management.io/description: ""
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    argocd.argoproj.io/sync-wave: "10"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+  name: ztp-argocd-plugins-installer
+  namespace: local-cluster
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: ztp-argocd-plugins-cp
+        spec:
+          remediationAction: enforce
+          severity: high
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: argoproj.io/v1beta1
+                kind: ArgoCD
+                metadata:
+                  name: openshift-gitops
+                  namespace: openshift-gitops
+                spec:
+                  controller:
+                    resources:
+                      limits:
+                        cpu: "16"
+                        memory: 32Gi
+                      requests:
+                        cpu: "1"
+                        memory: 2Gi
+                  kustomizeBuildOptions: "--enable-alpha-plugins"
+                  repo:
+                    resources:
+                      limits:
+                        cpu: "8"
+                        memory: "16Gi"
+                      requests:
+                        cpu: "1"
+                        memory: "2Gi"
+                    volumes:
+                      - name: "kustomize"
+                        emptyDir: {}
+                    initContainers:
+                      - name: "kustomize-plugin"
+                        command:
+                          - "/exportkustomize.sh"
+                        args:
+                          - "/.config"
+                        imagePullPolicy: "Always"
+                        volumeMounts:
+                          - name: "kustomize"
+                            mountPath: "/.config"
+                        terminationMessagePolicy: "File"
+                        terminationMessagePath: "/dev/termination-log"
+                        image: "registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.18"
+                      - name: "policy-generator-install"
+                        image: "registry.redhat.io/rhacm2/multicluster-operators-subscription-rhel9:v2.11"
```

Comment:
The main branch is currently aligned with 4.19 and using ACM 2.13 (pending release of 2.14).